concent_api/core/models.py

```diff
@@ -739,7 +739,7 @@ class DepositClaim(Model):
     payer_deposit_account = ForeignKey(DepositAccount)
     payee_ethereum_address = CharField(max_length=ETHEREUM_ADDRESS_LENGTH)
     concent_use_case = IntegerField()
-    amount = IntegerField()
+    amount = DecimalField(max_digits=78, decimal_places=0)
```

Comment:
We should have tests for all this model changes. Please add tests which will test maximum possible value for this changed fields

Reply:
Done